locales/en/index.yml

```diff
@@ -3214,7 +3214,7 @@ features:
       titleRight: Page {{currentPage}} of {{totalPages}}
     shareDataScreen:
       title: "Do you share this data?"
-      content: "To complete the signing process, you must share this data with the signature provider."
+      content: "They will be shared with the digital signature provider. If you cancel, you will not be able to complete the signing."
```

Comment:
Isn't "**this** data" singular? Why the use of "they" instead of "it"?

Reply:
Yep. Maybe a more right translation should be
"Do you want to share these data?" - data is a plural form of datum. 
"These will be shared with...."
What do you think about? @shadowsheep1

Reply:
Yup, much better.

Reply:
Maybe @thisisjp or @AliceAzzolini could give us an hand about it?

Reply:
Let's leave it as it is now — we'll review/translate the whole flow all at once